packages/remix/src/client/performance.tsx

```diff
@@ -175,3 +237,23 @@ export function withSentry<P extends Record<string, unknown>, R extends React.Co
   // will break advanced type inference done by react router params
   return SentryRoot;
 }
+
+export function setGlobals({
+  useEffect,
+  useLocation,
+  useMatches,
+  startTransactionOnLocationChange,
```

Comment:
l: Can we rename this to `instrumentNavigation` to align more with the new API? (not sure if we'd break something but looks like this is a new function)